Markup adjustment

BrBrRoman opened the pull request:
- Текст выровнен по ширине.
- Блок «Используемые технологии» разбит в маркированный список. Названия технологий выделены полужирным стилем.
- Увеличен кегль текста.
- Уменьшен размер зарисовок.
- Зарисовки страниц перенесены к их описанию и отцентрованы.
- Названия страниц и форм выделены полужирным стилем.
- Исправлены орфографические ошибки в словах «разбалловка» и «корректная».

Commit:
Markup adjustment

Commit:
Add technical_details in Markdown format

ziontab commented:
1) Можете ли вы внести правки в исходный html?
2) Как вы конвертировали из html файлов в md? Руками или может быть скриптом? Просто остальные html файлы нужно тоже сконвертировать, и желательно скриптом, чтобы иметь один первоисточник данных. Чтобы при изменение не нужно было править 2 места (а только принтить pdf и запускать скрипт для конверта в md например)
3) Можете ли Вы переделать остальные pdf под единый формат, как вы сделали для techical_details.pdf? Или вы принтили его по md документу готовому?

Commit:
Add tasks in Markdown format

Commit:
move files to "files" folder

Commit:
move files to 'files' folder

Commit:
Add tpark logo

Commit:
Add README file

Commit:
fill readme

Commit:
Правка заголовков второго уровня

В конце заголовков не должно быть знаков препинания

Commit:
fill readme file

Commit:
Скрыл URL адреса в блоке полезных ссылок

Commit:
Добавлены пустые строки перед списками

Для успешной конвертации md в pdf необходима пустая строка перед всеми списками

Commit:
Добавлены пустые строки перед списками

Commit:
Добавлены пустые строки перед списками

Commit:
Добавлены пустые строки перед списками

Commit:
Добавлены пустые строки перед списками

Commit:
Добавлены пустые строки перед списками

Commit:
Добавлены пустые строки перед списками

Commit:
Сделаны из markdown

Сделаны с помощью http://www.markdowntopdf.com/

Commit:
Правка орфографии

Commit:
Правка орфографии

Commit:
Правка орфографии

Commit:
Правка орфографии

Commit:
Правка орфографии

Commit:
Правка орфографии

Commit:
Правка орфографии

Commit:
Правка орфографии

Commit:
Из Markdown созданы HTML и PDF

BrBrRoman commented:
- Создал README.
- Вынес все файлы в отдельную папку.

> Можете ли вы внести правки в исходный html?

- Я не нашел утилит, которые могли бы правильно **пакетно** перевести markdown в HTML. 
Но Markdown можно перевести в HTML используя редактор Atom. Для нужного файла необходимо выбрать "Markdown Preview", а затем в появившемся окне выбрать "Save as HTML". 
- Я считаю, что HTML файлы не нужно хранить на GitHub, но они могут потребоваться для конвертации документов в PDF.


> Как вы конвертировали из html файлов в md? Руками или может быть скриптом?

- Правильнее было бы сделать это скриптом, но я сделал это руками.


> Чтобы при изменение не нужно было править 2 места (а только принтить pdf и запускать скрипт для конверта в md например)

- Я считаю, что документ необходимо создавать в разметке Markdown (т.к. это облегчённый язык разметки, созданный с целью написания удобного для правки текста), а затем переводить в HTML и PDF. Попробуйте открыть [техническое задание](https://github.com/BrBrRoman/tp-tasks/blob/master/files/markdown/technical_details.md) на мобильном устройстве. Заголовки и текст будут правильно перенесены, а изображения подогнаны по ширине экрана.


> Можете ли Вы переделать остальные pdf под единый формат, как вы сделали для techical_details.pdf? Или вы принтили его по md документу готовому?

- PDF из HTML можно получить с помощью LibreOffice. Пакетно обработать HTML можно с помощью команд `unoconv -f pdf -o ../pdf/ *.html` или `lowriter --convert-to pdf *.html --outdir ../pdf/`, **но** тогда в файлах task-2.pdf, task-3.pdf, task-6.pdf вставки кода отображаются неверно, а в файле technical_details.pdf описание зарисовок форм и сами зарисовки находятся на разных страницах. Данные файлы пришлось отдельно сконвертировать с помощью LibreOffice Writer с небольшими правками. 

Новые HTML и PDF уже находятся в [репозитории](https://github.com/BrBrRoman/tp-tasks). Из README скрыл ссылки на HTML файлы, т.к. через GitHub они открываются как код, а использовать HTML для оффлайна как-то странно. Но всё же эти файлы находятся в репозитории по адресу `/files/html/`

ziontab commented:
Роман, большое спасибо за проделанную работу!
